packages/astro/src/core/errors/errors-data.ts

```diff
@@ -92,6 +92,7 @@ export const PrerenderClientAddressNotAvailable = {
 export const StaticClientAddressNotAvailable = {
 	name: 'StaticClientAddressNotAvailable',
 	title: '`Astro.clientAddress` is not available in static mode.',
+	// TODO: Update this for the new static mode? I'm not sure this error can even still happen.
 	message:
 		"`Astro.clientAddress` is only available when using `output: 'server'` or `output: 'hybrid'`. Update your Astro config if you need SSR features.",
 	hint: 'See https://docs.astro.build/en/guides/server-side-rendering/ for more information on how to enable SSR.',
```

Comment:
I still need to fully check, but as far as I can tell, this error doesn't make sense anymore.